msteams-platform/tabs/how-to/using-fluid-msteam.md

```diff
@@ -0,0 +1,317 @@
+---
+title: Using Fluid with Microsoft Teams
+author: timtwang
+ms.author: mobajemu
+description: Tutorial for integrating Fluid-powered real-time collaboration features into a Microsoft Teams tab application
+ms.localizationpriority: medium
+ms.topic: conceptual
+---
+
+# Use Fluid with Microsoft Teams
+
+This tutorial helps to integrate Fluid-powered real-time collaboration features into a [Microsoft Teams tab application](/microsoftteams/platform/tabs/what-are-tabs). By the end of this tutorial you'll be able to integrate any Fluid-powered applications into Microsoft Teams and collaborate with others in real-time.
+
+In this section you'll learn the following concepts:
+
+1. How to integrate a Fluid client into a Microsoft Teams tab application.
+2. How to run and connect your Teams application to a Fluid service (Azure Fluid Relay).
+3. How to create and get Fluid Containers and pass them to a React component.
+
+For an example of how this recipe may be used to build a more complex application, check out the [Teams Fluid Hello World](https://github.com/microsoft/FluidExamples/tree/main/teams-fluid-hello-world) example in our FluidExamples repo.
+
+> [!NOTE]
+> This tutorial assumes that you are familiar with the [Fluid Framework Overview](https://fluidframework.com/docs/) and that you have completed the [QuickStart](https://fluidframework.com/docs/start/quick-start/). You should also be familiar with the basics of [React](https://reactjs.org/), [React Hooks](https://reactjs.org/docs/hooks-intro.html), and [Microsoft Teams Tabs](/microsoftteams/platform/tabs/what-are-tabs).
+
+## Create the project
+
+1. Open a Command Prompt and navigate to the parent folder where you want to create the project, for example, `/My Microsoft Teams Projects`.
+2. Create a vanilla Teams tab application by running the below command. See [the Microsoft Teams documentation](create-channel-group-tab.md#create-a-custom-channel-or-group-tab-with-nodejs)for more information on setting up Teams applications.
```

Comment:
Instead of "See the Microsoft Teams documentation" (since this article is now part of it), be more explicit about the specific article we're linking to